documentation/book/getting-started.adoc

```diff
@@ -9,12 +9,10 @@ The Docker images are available on {DockerRepository}.
 
 === Prerequisites
 
-An {ProductPlatformName} cluster is required to deploy {ProductLongName}. {ProductName} supports all kinds of clusters - from public and
-private clouds down to local deployments intended for development purposes. This guide expects that an {ProductPlatformName}
-cluster is available and the
-ifdef::Kubernetes[`kubectl` or]
-`oc` command line tools are installed and configured to connect
-to the running cluster.
+{ProductLongName} runs on {ProductPlatformName}.
+{ProductName} supports ifdef::Kubernetes[{KubernetesLongName} {KubernetesVersion}] or {OpenShiftLongName} {OpenShiftVersion}.
+{ProductName} works on all kinds of clusters - from public and private clouds down to local deployments intended for development.
+This guide expects that an {ProductPlatformName} cluster is available and the ifdef::Kubernetes[`kubectl` or] `oc` command line tools are installed and configured to connect to the running cluster.
```

Comment:
One more "an" left

Maybe we could even write a(n) here to stand in for both in place of "an" and "a". 

Or "This guide expects that at least one {ProductPlatformName} cluster is available"